src/components/PrivacyDisclaimer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+
+const Footer = styled.div`
+  width: 100vw;
+  overflow: hidden;
+  padding: 12px 40px;
+  display: flex;
+  border-top: 1px solid #efefef;
+  background-color: #fff;
+  align-items: center;
+  p {
+    flex: 1;
+    margin-right: 12px;
+    margin-bottom: 0px;
+  }
+  button.dismiss {
+    margin-right: 0px;
+  }
+  button {
+    margin-right: 12px;
+  }
+`;
+
+export const PRIVACY_ACCEPT = 'acceptPrivacy';
+
+class PrivacyDisclaimer extends React.Component {
+  componentDidMount() {
+    if (localStorage.getItem(PRIVACY_ACCEPT) === null) {
+      setTimeout(() => {
+        localStorage.setItem(PRIVACY_ACCEPT, 'true');
+        this.forceUpdate();
+      }, 30000);
+    } else if (navigator.doNotTrack === '1') {
+      // eslint-disable-next-line no-console
+      console.warn(
+        "Your browser's Do Not Track is turned on. Go to https://allaboutdnt.com/ to learn how you can turn it off."
+      );
+    }
+  }
+
+  render() {
+    if (
+      localStorage.getItem(PRIVACY_ACCEPT) === 'true' ||
+      localStorage.getItem(PRIVACY_ACCEPT) === 'false'
+    ) {
+      return null;
+    }
+    return (
+      <Footer>
+        <p>
+          Just a heads up that we are tracking your email and usage in
```

Comment:
No text yet, and since its just an internal power tool, I was not sure if we need to go through security/marketing approval :/ Input would greatly be appreciated

Reply:
I think "tracking your email" gives the wrong impression and sounds to negative/scary, the message should be more generic about tracking usage. The email-part is a technicality that the user shouldn't have to think about. 

We could just as well use a hash of the email or an internal user id to do the same tracking with the same privacy implications.

Reply:
It's still not gdpr compliant if we do the hash since we can still uniquely identify the user :( I couldn't find a great example online for what to say, I'll try and look around more. (since we're actually not doing sketchy stuff like selling their data)

Reply:
`Tracking your usage` would work? But check with ADI/OpSup!

Reply:
Sooo they dont have anything that indicates they are doing this... so maybe we should come up with our own standard?